phpBB/styles/prosilver/template/icons/iconify.html

```diff
@@ -0,0 +1,4 @@
+{% spaceless %}
+	<i class="iconify o-icon{{ CLASSES ? ' ' ~ CLASSES }}"{% if S_HIDDEN %}{% if TITLE %} title="{{ lang(TITLE) }}"{% endif %} aria-hidden="true"{% endif %} data-icon="fa:{{ ICON }}" data-inline="true"{{ ATTRIBUTES }}></i>
```

Comment:
`data-icon="fa:{{ ICON }}"` is incorrect the fa: is part of the `ICON` variable since you can use any icon from any library you could have fa:home, mdi:home, is:home, dash:home, mdl:home, etc...